src/client/pages/register/CreateUser.jsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { LOCAL_STORAGE, INT_LINK, USER_URL } from '../../config';
+
+import { doFetch } from '../../utils';
```

Comment:
doFetch?

Sounds like a Pokémon. Consider a clearer name.

If it makes http requests, conventionally use a name like `client`, and encapsulate behaviour accordingly: client.get(URL), etc